models/project/project.go

```diff
@@ -371,3 +308,22 @@ func (u *ProjectStruct) FetchGithub() (bool, error) {
 
 	return true, nil
 }
+
+// OautToken get oauth token in users
```

Comment:
__[golint]__
* [golint] comment on exported method ProjectStruct.OauthToken should be of the form "OauthToken ..."